Source/Core/Core/PowerPC/JitCommon/JitCache.cpp

```diff
@@ -36,6 +36,11 @@ static void ClearCacheThreadSafe(u64 userdata, s64 cyclesdata)
   JitInterface::ClearCache();
 }
 
+bool JitBlock::Overlap(u32 addr, u32 length)
```

Comment:
This can be a const member function. It should also probably indicate what is being overlapped. e.g `OverlapsThing()`